docs/source/environment_variables.rst

```diff
@@ -0,0 +1,664 @@
+
+`ONEFLOW_DATA_FILE_SYSTEM_TYPE <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/persistence/file_system.cpp#L172>`_
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+The disk type used by the dataset, supports hdfs(`CreateHadoopFS <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/persistence/file_system.cpp#L161>`_) and `CreateLocalFS <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/persistence/file_system.cpp#L153>`_. If use hdfs, you need to set ``ONEFLOW_DATA_FILE_SYSTE_HDFS_NAMENODE`` to initialize the hdfs node. `#5162 <https://github.com/Oneflow-Inc/oneflow/pull/5162>`_
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is empty, representing ``local``, and another optional value is ``hdfs``.
+
+`ONEFLOW_SNAPSHOT_FILE_SYSTEM_TYPE <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/persistence/file_system.cpp#L182>`_
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+The disk type useed when saving and loading models. It is the same as above, if using hdfs, ``ONEFLOW_SNAPSHOT_FILE_SYSTEM_HDFS_NAMENODE`` need to be used together. `5162 <https://github.com/Oneflow-Inc/oneflow/pull/5162>`_
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is empty, representing ``local``, and another optional value is ``hdfs``.
+
+`ONEFLOW_PERSISTENT_IN_STREAM_BUFFER_SIZE_BYTES <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/persistence/persistent_in_stream.cpp#L30>`_
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Represents the size of each read from disk. `#5162 <https://github.com/Oneflow-Inc/oneflow/pull/5162>`_
+
+Values accepted
+^^^^^^^^^^^^^^^
+The default value is empty. If an invalid string or negative number is entered, the default value would be ``32 * 1024; // 32KB``.
+
+`ONEFLOW_DECODER_ENABLE_NVJPEG_HARDWARE_ACCELERATION <https://github.com/Oneflow-Inc/oneflow/blob/v0.5.0rc1/oneflow/core/kernel/image_decoder_random_crop_resize_kernel.cpp#L252>`_
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+``NVJPEG_VER_MAJOR`` need to be bigger than 11. It can accelerate nvjpeg hardware, warm up jpeg decoder and hw_jpeg decoder, `#5851 <https://github.com/Oneflow-Inc/oneflow/pull/5851>`_. 
```

Comment:
```suggestion
``NVJPEG_VER_MAJOR`` need to be bigger than ``11``. It can accelerate nvjpeg hardware, warm up jpeg decoder and hw_jpeg decoder, `#5851 <https://github.com/Oneflow-Inc/oneflow/pull/5851>`_. 
```

Reply:
已改